docs/how-to/default-profile.md

```diff
@@ -96,7 +96,7 @@ rm /tmp/ipfs-ds-convert.tar.gz
 
 ### Windows
 
-For Windows, we will download the zip file, extract it and then add the path to `ipfs-ds-convert.exe` to our environment path
+For Windows, you download the zip file, extract it and then add the path to `ipfs-ds-convert.exe` to your environment path:
```

Comment:
```suggestion
Download the zip file, extract it and then add the path to `ipfs-ds-convert.exe` to your environment path:
```

Let's turn this into an instruction rather than a cometary.